spline-shadow/migrator-tool/src/test/scala/za/co/absa/spline/migrator/MigratorToolSpec.scala

```diff
@@ -28,14 +32,20 @@ import scala.util.Try
 @Ignore
 class MigratorToolSpec extends FunSpec {
 
-  private val arangoUri = "http://root:root@localhost:8529/_system"
+
+  private val arangoUri = "http://root:root@localhost:8529/unit-test"
+  private val mongoUri = "mongodb://localhost:27017/migration-test"
 
   describe("migration tool test") {
     it("migrate from mongo to arango") {
-      val database = Database(new URI(arangoUri))
-      Try(awaitForever(database.delete(true)))
-      awaitForever(database.init(force = true))
-      Await.result(MigratorTool.migrate(new MigratorConfig("mongodb://localhost:27017/migration-test", arangoConnectionUrl = arangoUri, batchSize = 10, batchesMax = 1)), Duration.Inf)
+      val db = ArangoFactory.create(new URI(arangoUri))
+      Try(db.drop())
```

Comment:
It is there to ignore deletion failure if the database doesn't exist.

Reply:
There are must be some treatment otherwise the meaning of empty `Try()` expression is just too opaque.
You may create a common helper method, or implicit method to mixin into the ArangoDatabase class and use it to drop a db silently. E.g.:

    object ArangoDBImplicits {
        implicit class ArangoDBWrapper(db: ArangoDatabase) {
            def dropSilently: Boolean = 
                Try(db.drop) recover {
                    case e: ArangoDBException if e.getResponseCode == ERROR_ARANGO_DATABASE_NOT_FOUND => true
                }
        }
    }